src/server/console/logind_console_services.cpp

```diff
@@ -175,6 +250,31 @@ mir::LogindConsoleServices::LogindConsoleServices()
         "notify::state",
         G_CALLBACK(&LogindConsoleServices::on_state_change),
         this);
+    g_signal_connect(
+        G_OBJECT(session_proxy.get()),
+        "pause-device",
+        G_CALLBACK(&LogindConsoleServices::on_pause_device),
+        this);
+
+#ifdef MIR_GDBUS_SIGNALS_SUPPORT_FDS
```

Comment:
This is true; there is no platform where GDBus signals support fds, because I haven't written the patch yet.

I kept it in there because (a) it'll help in validating that patch, and (b) it's documenting an otherwise-perplexing difference (why do we hook up signal handlers for everything else, but use this weird message-filter API for the Activated signal?)